frontend/src/components/common/button/button.tsx

```diff
@@ -1,12 +1,15 @@
 import { FC } from 'common/types/types';
+import styles from './button.module.scss';
 
 type Props = {
```

Comment:
```suggestion
type Props = React.ComponentPropsWithoutRef<'button'>;
```

Reply:
Or just import { ComponentPropsWithoutRef } from 'react'

Reply:
Actually, we don't want this. let's describe props separately